cmake/CMakeLists.txt

```diff
@@ -159,6 +159,10 @@ else (MSVC)
   set(LIB_PREFIX)
 endif (MSVC)
 
+if (UNICODE)
+  add_definitions(-DUNICODE -D_UNICODE)
```

Comment:
I see. Can you name the option the same way as other options? I.e., protobuf_UNICODE